src/Altinn.AccessManagement/Program.cs

```diff
@@ -237,8 +237,19 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     services.AddSingleton<IResourceAdministrationPoint, ResourceAdministrationPoint>();
     services.AddSingleton<IPolicyRepository, PolicyRepository>();
     services.AddSingleton<IResourceRegistryClient, ResourceRegistryClient>();
-    services.AddSingleton<IDelegationMetadataRepository, DelegationMetadataRepository>();
-    services.AddSingleton<IResourceMetadataRepository, ResourceMetadataRepository>();
+
+    if (config.GetValue<bool>("AccessMgmt:General:UseNewQueryRepo"))
```

Comment:
I don't see other config changes for setup or adding this to feature management in appsettings.development.json for local dev.

Reply:
Correct. On-The-Fly feature toggle with Azure App Config is moved to another PR to keep things simple. The setting needs to be added with env variables for the time beeing. It will default to `false` if the setting is missing. I've added the setting to `appsettings.development.json` under Feature Management.

Reply:
I'm testing a possible way to have this implemented with feature flags.